numpy/core/src/multiarray/compiled_base.h

```diff
@@ -1,6 +1,21 @@
 #ifndef _NPY_PRIVATE__COMPILED_BASE_H_
 #define _NPY_PRIVATE__COMPILED_BASE_H_
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+#include <Python.h>
+#include <structmember.h>
+#include <string.h>
+
+#define _MULTIARRAYMODULE
+#include "numpy/arrayobject.h"
+#include "numpy/npy_3kcompat.h"
 #include <numpy/ndarraytypes.h>
+#include "numpy/npy_math.h"
+#include "npy_config.h"
+#include "templ_common.h" /* for npy_mul_with_overflow_intp */
+#include "lowlevel_strided_loops.h" /* for npy_bswap8 */
+#include "alloc.h"
+#include "ctors.h"
+#include "common.h"
```

Comment:
Is there any reason these needed to be moved? They'd be better back in `compiled_base.c` where they were before